crates/matrix-sdk-base/src/sliding_sync.rs

```diff
@@ -585,7 +585,9 @@ async fn cache_latest_events(
     for event in events.iter().rev() {
         if let Ok(timeline_event) = event.event.deserialize() {
             match is_suitable_for_latest_event(&timeline_event) {
-                PossibleLatestEvent::YesRoomMessage(_) | PossibleLatestEvent::YesPoll(_) => {
+                PossibleLatestEvent::YesRoomMessage(_)
+                | PossibleLatestEvent::YesPoll(_)
+                | PossibleLatestEvent::YesCallInvite(_) => {
                     // m.room.message or m.poll.start - we found one! Store it.
```

Comment:
nit: update the comment here to make it more generic, maybe? no need to re-precise the matrix type here, IMO